File: armstrongNumbers.js
function armstrongNumbers(input){
  let number = input.toString()
  let multiplyArray = []
  
for(let i = 0; i < number.length; i++){
  multiplyArray.push(number[i] ** 3)
}
let answer = multiplyArray.reduce(function(a, b){
  return a + b;
}, 0);
if(answer === input){
  console.log(`${input} is an armstrong number`)
} else {
  console.log(`${input} is not an armstrong number`)
}
}

armstrongNumbers(153)

File: armstrong_numbers.py
def armstrong_numbers(input):
  number = str(input)
  multiply_array = []

  for i in map(int, number):
    multiply_array.append(i ** 3)
    
  answer = sum(multiply_array)
  if(answer == input):
    print(f"{input} is an armstrong number")
  else:
    print(f"{input} is not an armstrong number")

armstrong_numbers(153)
